src/app/app.component.ts

```diff
@@ -67,7 +71,16 @@ export class AppComponent {
     this.activatedRoute.params.subscribe(() => {
```

Comment:
I see several subscriptions in this particular class, but they don't appear to be destroyed? I believe we need to unsubscribe.